test/router.test.ts

```diff
@@ -0,0 +1,212 @@
+import invariant from 'tiny-invariant'
+import { ChainId, CurrencyAmount, ETHER, Pair, Percent, Route, Router, Token, TokenAmount, Trade, WETH } from '../src'
+import JSBI from 'jsbi'
+
+function checkDeadline(deadline: string[] | string): void {
+  expect(typeof deadline).toBe('string')
+  invariant(typeof deadline === 'string')
```

Comment:
don't need this

Reply:
toBe doesn't work as a type assertion but invariant is

Reply:
oh got it